x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/rule_preview/preview_table_cell_renderer.tsx

```diff
@@ -5,22 +5,23 @@
  * 2.0.
  */
 
+import type { ComponentProps } from 'react';
 import React from 'react';
 import type { EuiDataGridCellValueElementProps } from '@elastic/eui';
 import { TableId } from '@kbn/securitysolution-data-table';
 import type { CellValueElementProps } from '../../../../../common/types';
 import { SourcererScopeName } from '../../../../sourcerer/store/model';
-import { RenderCellValue } from '../../../../detections/configurations/security_solution_detections';
+import { CellValue } from '../../../../detections/configurations/security_solution_detections';
 
 export const PreviewRenderCellValue: React.FC<
   EuiDataGridCellValueElementProps & CellValueElementProps
 > = (props) => {
   return (
-    <RenderCellValue
-      {...props}
+    <CellValue
+      {...(props as unknown as ComponentProps<typeof CellValue>)}
```

Comment:
nit: Is it possible to avoid the cast?

Reply:
They're using the same cell renderer out of the table context so some props are missing. This is a temporary solution while waiting to share this with Security and see what they prefer (i.e. splitting the cell component in two)